hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/DefaultRecordStore.java

```diff
@@ -1464,4 +1525,9 @@ private void clearLockStore() {
             lockService.clearLockStore(partitionId, namespace);
         }
     }
+
+    @Override
+    public Set<MapOperation> getOffloadedOperations() {
+        return offloadedOperations;
```

Comment:
This is simple LinkedHashMap Is this accessed from a single thread only? If so we should document that.